### PR TITLE
Create SUPPORT.md

### PR DESCRIPTION
**Pull request by HurricanKai**
Welp, GitHub adding these new files all the time, so here we go filling out GitHub files.

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Getting Support
+For support, please either use [GitHub Discussions](https://github.com/dotnet/Silk.NET/discussions) or contact as via [Discord](https://discord.com/invite/DTHHXRt)
```
Comment (Perksey) [inline .github/SUPPORT.md:2]:
```suggestion
For support, please either use [GitHub Discussions](https://github.com/dotnet/Silk.NET/discussions) or chat to us on [Discord](https://discord.com/invite/DTHHXRt).
```

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+
+<a href="https://discord.com/invite/DTHHXRt"><img src="https://discordapp.com/api/guilds/521092042781229087/widget.png?style=banner2" alt="Banner image of the Silk.NET Discord server"/></a>
```
Comment (Perksey) [inline .github/SUPPORT.md:5]: Can we defer Discord's official badges, as these brought a lot of "Sex Dating" bots into the OpenTK server when we used them over there.

**Review by Perksey — CHANGES_REQUESTED** [review]

File: .github/SUPPORT.md (hunk 2 above)
Comment (HurricanKai) [reply .github/SUPPORT.md:5]: What do you mean?

Comment (Perksey) [reply .github/SUPPORT.md:5]: As in we had a lot of bots join the server using the link that posted links entitled "Sex Dating"

Comment (HurricanKai) [reply .github/SUPPORT.md:5]: Hmm, I mean we advertise this everywhere anyways, so why not here as well?

Comment (Perksey) [reply .github/SUPPORT.md:5]: The discord link is fine, but it's something about the official widget.png in particular that attracts them.

Comment (Perksey) [reply .github/SUPPORT.md:5]:
```suggestion
<a href="https://discord.com/invite/DTHHXRt"><img src="https://img.shields.io/badge/chat%20on-discord-7289DA" alt="Banner image of the Silk.NET Discord server"/></a>
```

**Review by Perksey — APPROVED** [review]

**Pull request merged**